src/aarch64/logic-aarch64.cc

```diff
@@ -6799,18 +6802,21 @@ LogicVRegister Simulator::fexpa(VectorFormat vform,
 
   if (lane_size == kHRegSize) {
     index_highbit = 4;
-    VIXL_ASSERT(ArrayLength(fexpa_coeff16) == (1U << (index_highbit + 1)));
+    VIXL_ASSERT(ArrayLength(fexpa_coeff16) ==
+                (UINT64_C(1) << (index_highbit + 1)));
```

Comment:
Can you use uint64_t{1} style for this? I realise there are others in the code, but I've a patch to remove them, and I'd like to avoid adding new ones.

Reply:
I will get to this soon

Reply:
Changes applied (to the ones I've added)